cmd/buildah/push.go

```diff
@@ -88,9 +88,20 @@ func pushCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
+
 	dest, err := alltransports.ParseImageName(destSpec)
+	// add the docker:// transport to see if they neglected it.
 	if err != nil {
-		return err
+		if !strings.Contains(destSpec, "://") {
```

Comment:
Reverse the error
if err strings.Container(destSpec,"://") {
    return err
}